### PR TITLE
feat: Add new tokens to OutOfStock

### PR DESCRIPTION
**Pull request by ArthurTriis1**
## What's the purpose of this pull request?
This PR applies the new [Theming structure](https://github.com/vtex-sites/base.store/pull/407) to `OutOfStck`: new scoped tokens linked to global ones.

Additionally, I adjusted the OutOfStock's style.

## How does it work?
This PR uses local variables to parameterize the `OutOfStock` properties, then connects these scoped tokens to the global ones.

Screen Shot 2022-06-06 at 14 34 16

### Title

<table><thead><tr><th>Local token</th><th>Default value/Global token linked</th></tr></thead><tbody><tr><td><code class="css-1npuvnl">--fs-out-of-stock-title-margin-bottom</code></td><td><code class="css-1npuvnl">var(--fs-spacing-0)</code></td></tr><tr><td><code class="css-1npuvnl">--fs-out-of-stock-title-line-height</code></td><td><code class="css-1npuvnl">1.15</code></td></tr><tr><td><code class="css-1npuvnl">--fs-out-of-stock-title-size</code></td><td><code class="css-1npuvnl">var(--fs-text-size-3)</code></td></tr><tr><td><code class="css-1npuvnl">--fs-out-of-stock-title-weight</code></td><td><code class="css-1npuvnl">var(--fs-text-weight-bold)</code></td></tr></tbody></table>

### Message

<table><thead><tr><th>Local token</th><th>Default value/Global token linked</th></tr></thead><tbody><tr><td><code class="css-1npuvnl">--fs-out-of-stock-message-column-gap</code></td><td><code class="css-1npuvnl">var(--fs-spacing-0)</code></td></tr><tr><td><code class="css-1npuvnl">--fs-out-of-stock-message-margin-bottom</code></td><td><code class="css-1npuvnl">var(--fs-spacing-3)</code></td></tr><tr><td><code class="css-1npuvnl">--fs-out-of-stock-message-line-height</code></td><td><code class="css-1npuvnl">1.15</code></td></tr><tr><td><code class="css-1npuvnl">--fs-out-of-stock-message-size</code></td><td><code class="css-1npuvnl">var(--fs-text-size-base)</code></td></tr><tr><td><code class="css-1npuvnl">--fs-out-of-stock-message-weight</code></td><td><code class="css-1npuvnl">var(--fs-text-weight-regular)</code></td></tr></tbody></table>

### Button

<table><thead><tr><th>Local token</th><th>Default value/Global token linked</th></tr></thead><tbody><tr><td><code class="css-1npuvnl">--fs-out-of-stock-button-width</code></td><td><code class="css-1npuvnl">100%</code></td></tr><tr><td><code class="css-1npuvnl">--fs-out-of-stock-button-margin-top</code></td><td><code class="css-1npuvnl">var(--fs-spacing-3)</code></td></tr></tbody></table>

## Checklist
- [x] Added an entry in the `CHANGELOG.md` at the beginning of its due section. [The latest version should comes first](https://keepachangelog.com/en/1.0.0/#:~:text=The%20latest%20version%20comes%20first.).
- [x] Added the PR number with the PR link at the entry in the `CHANGELOG.md`. E.g., *New items in the `pull_request_template.md` ([#4](https://github.com/vtex-sites/nextjs.store/pull/4))* 
- [x] Updated the Storybook - *if applicable*.
- [x] Added a label according to the PR goal - `Breaking change`, `Enhancement`, `Bug` or `Chore`.
- [x] Added the component, hook, or pathname in-between backticks (``) *- If applicable*. E.g., *`ComponentName` component*.

### CODE REVIEW

**Review by filipewl — APPROVED** [review]
Looks good to me! It seems we missed one minor update to the NextJS version though.

**Review by hellofanny — APPROVED** [review]
Looks good to me! 🚀 
I also noticed something:

We removed the `testId` here but it seems like we are still displaying it on [NextJS](https://nextjs-store-storybook-faststore.vercel.app/?path=/docs/organisms-outofstock--out-of-stock) 😕 

image

File: src/components/product/OutOfStock/out-of-stock.module.scss [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+  display: flex;
+  flex-direction: column;
+
+  [data-fs-out-of-stock-title] {
+    margin-bottom: var(--fs-out-of-stock-title-margin-bottom);
+    font-size: var(--fs-out-of-stock-title-size);
+    font-weight: var(--fs-out-of-stock-title-weight);
+    line-height: var(--fs-out-of-stock-title-line-height);
+    color: var(--fs-out-of-stock-title-color);
```
[the remark this answers is not in the document]
Comment (hellofanny) [reply src/components/product/OutOfStock/out-of-stock.module.scss:39]: Me too! Little things to updated on NextJS version ~

**Review by eduardoformiga — APPROVED** [review]
LGTM!

Could you please apply the suggestions mentioned previously?

**Comment by ArthurTriis1** [comment]
> Looks good to me! 🚀 I also noticed something:
> 
> We removed the `testId` here but it seems like we are still displaying it on [NextJS](https://nextjs-store-storybook-faststore.vercel.app/?path=/docs/organisms-outofstock--out-of-stock) 😕
> 
> image

I think it wasn't updated yet. In local, the testId don't appear.

**Pull request merged**